runhouse/rns/hardware/cluster.py

```diff
@@ -659,3 +660,54 @@ def remove_conda_env(
         """Remove conda env from the cluster."""
         env_name = env if isinstance(env, str) else env.env_name
         self.run([f"conda env remove -n {env_name}"])
+
+
+# Cluster factory method
+def cluster(
+    name: str,
+    ips: List[str] = None,
+    ssh_creds: Optional[dict] = None,
+    dryrun: bool = False,
+    **kwargs,
+) -> Cluster:
+    """
+    Builds an instance of :class:`Cluster`.
+
+    Args:
+        name (str): Name for the cluster, to re-use later on.
+        ips (List[str], optional): List of IP addresses for the BYO cluster.
+        ssh_creds (dict, optional): Dictionary mapping SSH credentials.
+            Example: ``ssh_creds={'ssh_user': '...', 'ssh_private_key':'<path_to_key>'}``
+        dryrun (bool): Whether to create the Cluster if it doesn't exist, or load a Cluster object as a dryrun.
+            (Default: ``False``)
+
+    Returns:
+        Cluster: The resulting cluster.
+
+    Example:
+        >>> # BYO Cluster
+        >>> gpu = rh.autocluster(ips=['<ip of the cluster>'],
```

Comment:
```suggestion
        >>> gpu = rh.cluster(ips=['<ip of the cluster>'],
```

Reply:
If I have an on demand cluster saved in RNS I can still re-load it with `rh.cluster(name="name", dryrun=True)`, and will get back an `OnDemandCluster` since we resolve its cluster type in the `from_config` method. 

I changed it above to be more explicit, but do you think it's confusing that we now have 2 ways to reload an on demand cluster (since you can now also reload it using `rh.autocluster`)?